Assets/MixedRealityToolkit/InputSystem/Focus/TeleportTarget.cs

```diff
@@ -8,34 +8,19 @@ namespace Microsoft.MixedReality.Toolkit.InputSystem.Focus
 {
     public class TeleportTarget : InteractableHighlight, ITeleportTarget
     {
-        [Header("Hot Spot Options")]
         [SerializeField]
-        private bool overrideOrientation;
+        [Header("Hot Spot Options")]
```

Comment:
Why reorder the Header and SerializeField attributes? The header applies to more than just this one field, correct?

Reply:
It doesn't matter the order. I'm a bit strange about having lines grow. haha.  In reality if we want to do it the professional way we should remove the header attribute and do a custom inspector.